storage/mysql/log_storage_test.go

```diff
@@ -1158,3 +1158,17 @@ func leafInBatch(leaf *trillian.LogLeaf, batch []*trillian.LogLeaf) bool {
 
 	return false
 }
+
+// byLeafIdentityHash allows sorting of leaves by their identity hash, so DB
```

Comment:
Can this be removed altogether?

Reply:
If not, could we replace it with `sort.Slice(s, func(i, j int) { return bytes.Compare(s[i].LeafIdentityHash, s[j].LeafIdentityHash) == -1 }`?